<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html lang="en">
<head>
<title>JabRef References output</title>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<script type="text/javascript">
<!--
// QuickSearch script for JabRef HTML export 
// Version: 2.0
//
// Copyright (c) 2006-2008, Mark Schenk
//
// This software is distributed under a Creative Commons Attribution 3.0 License
// http://creativecommons.org/licenses/by/3.0/

// Some features:
// + optionally searches Abstracts and Reviews
// + allows RegExp searches
//   e.g. to search for entries between 1980 and 1989, type:  198[0-9]
//   e.g. for any entry ending with 'symmetry', type:  symmetry$
//   e.g. for all reftypes that are books: ^book$, or ^article$
//   e.g. for entries by either John or Doe, type john|doe
// + easy toggling of Abstract/Review/BibTeX

// Search settings
var searchAbstract = true;
var searchReview = true;

// Speed optimisation introduced some esoteric problems with certain RegExp searches
// e.g. if the previous search is 200[-7] and the next search is 200[4-7] then the search doesn't work properly until the next 'keyup'
// hence the searchOpt can be turned off for RegExp adepts
var searchOpt = true;

if (window.addEventListener) {
	window.addEventListener("load",initSearch,false); }
else if (window.attachEvent) {
	window.attachEvent("onload", initSearch); }

function initSearch() {
	// basic object detection
	if(!document.getElementById || !document.getElementsByTagName) { return; }
	if (!document.getElementById('qstable')||!document.getElementById('qs')) { return; }

	// find QS table and appropriate rows
	searchTable = document.getElementById('qstable');
	var allRows = searchTable.getElementsByTagName('tbody')[0].getElementsByTagName('tr');

	// split all rows into entryRows and infoRows (e.g. abstract, review, bibtex)
	entryRows = new Array();
	infoRows = new Array(); absRows = new Array(); revRows = new Array();

	for (var i=0, k=0, j=0; i<allRows.length;i++) {
		if (allRows[i].className.match(/entry/)) {
			entryRows[j++] = allRows[i];
		} else {
			infoRows[k++] = allRows[i];
			// check for abstract/review
			if (allRows[i].className.match(/abstract/)) {
				absRows.push(allRows[i]);
			} else if (allRows[i].className.match(/review/)) {
				revRows.push(allRows[i]);
			}
		}
	}

	//number of entries and rows
	numRows = allRows.length;
	numEntries = entryRows.length;
	numInfo = infoRows.length;
	numAbs = absRows.length;
	numRev = revRows.length;

	//find the query field
	qsfield = document.getElementById('qsfield');

	// previous search term; used for speed optimisation
	prevSearch = '';

	//find statistics location
	stats = document.getElementById('stat');
	setStatistics(-1);

	// creates the appropriate search settings
	createQSettingsDialog();

	// shows the searchfield
	document.getElementById('qs').style.display = 'block';
	document.getElementById('qsfield').onkeyup = testEvent;
}

function quickSearch(tInput){

	 if (tInput.value.length == 0) {
		showAll();
		setStatistics(-1);
		qsfield.className = '';
		return;
	} else {
		// only search for valid RegExp
		try {
			var searchText = new RegExp(tInput.value,"i")
			closeAllInfo();
			qsfield.className = '';
		}
		catch(err) {
			prevSearch = tInput.value;
			qsfield.className = 'invalidsearch';
			return;
		}
	}
	
	// count number of hits
	var hits = 0;

	// start looping through all entry rows
	for (var i = 0; cRow = entryRows[i]; i++){

		// only show search the cells if it isn't already hidden OR if the search term is getting shorter, then search all
		// some further optimisation is possible: if the search string is getting shorter, and the row is already visible, skip it. Then be careful with hits!
		if(!searchOpt || cRow.className.indexOf('noshow')==-1 || tInput.value.length <= prevSearch.length){
			var found = false; 

			var inCells = cRow.getElementsByTagName('td');
			var numCols = inCells.length;
				
			for (var j=0; j<numCols; j++) {
				cCell = inCells[j];
				var t = cCell.innerText?cCell.innerText:getTextContent(cCell);
				if (t.search(searchText) != -1){ 
					found=true; 
					break;
				} 
			}

			// look for further hits in Abstract and Review
			if(!found) {
				var articleid = cRow.id;
				if(searchAbstract && (abs = document.getElementById('abs_'+articleid))) {
					if (getTextContent(abs).search(searchText) != -1){ found=true; } 
				}
				if(searchReview && (rev = document.getElementById('rev_'+articleid))) {
					if (getTextContent(rev).search(searchText) != -1){ found=true; } 
				}
			}
			
			if(found) {
				cRow.className = 'entry show';
				hits++;
			} else {
				cRow.className = 'entry noshow';
			}
		}
	}

	// update statistics
	setStatistics(hits)
	
	// set previous search value
	prevSearch = tInput.value;
}

function toggleInfo(articleid,info) {

	var entry = document.getElementById(articleid);
	var abs = document.getElementById('abs_'+articleid);
	var rev = document.getElementById('rev_'+articleid);
	var bib = document.getElementById('bib_'+articleid);

	// Get the abstracts/reviews/bibtext in the right location
	// in unsorted tables this is always the case, but in sorted tables it is necessary. 
	// Start moving in reverse order, so we get: entry, abstract,review,bibtex
	if (searchTable.className.indexOf('sortable') != -1) {
		if(bib) { entry.parentNode.insertBefore(bib,entry.nextSibling); }
		if(rev) { entry.parentNode.insertBefore(rev,entry.nextSibling); }
		if(abs) { entry.parentNode.insertBefore(abs,entry.nextSibling); }
	}

	if (abs && info == 'abstract') {
		if(abs.className.indexOf('abstract') != -1) {
		abs.className.indexOf('noshow') == -1?abs.className = 'abstract noshow':abs.className = 'abstract';
		}
	} else if (rev && info == 'review') {
		if(rev.className.indexOf('review') != -1) {
		rev.className.indexOf('noshow') == -1?rev.className = 'review noshow':rev.className = 'review';
		}
	} else if (bib && info == 'bibtex') {
		if(bib.className.indexOf('bibtex') != -1) {
		bib.className.indexOf('noshow') == -1?bib.className = 'bibtex noshow':bib.className = 'bibtex';
		}		
	} else { 
		return;
	}
	
	// check if one or the other is available
	var revshow = false;
	var absshow = false;
	var bibshow = false;
	(abs && abs.className.indexOf('noshow') == -1)? absshow = true: absshow = false;
	(rev && rev.className.indexOf('noshow') == -1)? revshow = true: revshow = false;	
	(bib && bib.className == 'bibtex')? bibshow = true: bibshow = false;
	
	// highlight original entry
	if(entry) {
		if (revshow || absshow || bibshow) {
		entry.className = 'entry highlight show';
		} else {
		entry.className = 'entry show';
		}		
	}
	
	// When there's a combination of abstract/review/bibtex showing, need to add class for correct styling
	if(absshow) {
		(revshow||bibshow)?abs.className = 'abstract nextshow':abs.className = 'abstract';
	} 
	if (revshow) {
		bibshow?rev.className = 'review nextshow': rev.className = 'review';
	}
	
}

function setStatistics (hits) {
	if(hits < 0) { hits=numEntries; }
	if(stats) { stats.firstChild.data = hits + '/' + numEntries}
}

function getTextContent(node) {
	// Function written by Arve Bersvendsen
	// http://www.virtuelvis.com
	
	if (node.nodeType == 3) {
	return node.nodeValue;
	} // text node
	if (node.nodeType == 1) { // element node
	var text = [];
	for (var chld = node.firstChild;chld;chld=chld.nextSibling) {
		text.push(getTextContent(chld));
	}
	return text.join("");
	} return ""; // some other node, won't contain text nodes.
}

function showAll(){
	// first close all abstracts, reviews, etc.
	closeAllInfo();

	for (var i = 0; i < numEntries; i++){
		entryRows[i].className = 'entry show'; 
	}
}

function closeAllInfo(){
	for (var i=0; i < numInfo; i++){
		if (infoRows[i].className.indexOf('noshow') ==-1) {
			infoRows[i].className = infoRows[i].className + ' noshow';
		}
	}
}

function testEvent(e){
	if (!e) var e = window.event;
	quickSearch(this);
}

function clearQS() {
	qsfield.value = '';
	quickSearch(qsfield);
}

function redoQS(){
	showAll();
	quickSearch(qsfield);
}

// Create Search Settings

function toggleQSettingsDialog() {

	var qssettings = document.getElementById('qssettings');
	
	if(qssettings.className.indexOf('active')==-1) {
		qssettings.className = 'active';

		if(absCheckBox && searchAbstract == true) { absCheckBox.checked = 'checked'; }
		if(revCheckBox && searchReview == true) { revCheckBox.checked = 'checked'; }

	} else {
		qssettings.className= '';
	}
}

function createQSettingsDialog(){
	var qssettingslist = document.getElementById('qssettings').getElementsByTagName('ul')[0];
	
	if(numAbs!=0) {
		var x = document.createElement('input');
		x.id = "searchAbs";
		x.type = "checkbox";
		x.onclick = toggleQSetting;
		var y = qssettingslist.appendChild(document.createElement('li')).appendChild(document.createElement('label'));
		y.appendChild(x);
		y.appendChild(document.createTextNode('search abstracts'));		
	}
	if(numRev!=0) {
		var x = document.createElement('input');
		x.id = "searchRev";
		x.type = "checkbox";		
		x.onclick = toggleQSetting;
		var y = qssettingslist.appendChild(document.createElement('li')).appendChild(document.createElement('label'));		
		y.appendChild(x);		
		y.appendChild(document.createTextNode('search reviews'));
	}
		
	// global variables
	absCheckBox = document.getElementById('searchAbs');
	revCheckBox = document.getElementById('searchRev');
	
	// show search settings
	if(absCheckBox||revCheckBox) {
		document.getElementById('qssettings').style.display = 'block';
	}
}

function toggleQSetting() {
	if(this.id=='searchAbs') { searchAbstract = !searchAbstract; }
	if(this.id=='searchRev') { searchReview = !searchReview; }
	redoQS()
}
-->
</script>
<script type="text/javascript">
<!--
// Sort Table Script
// Version: 1.1
//
// Copyright (c) 2006-2008, Mark Schenk
//
// This software is distributed under a Creative Commons Attribution 3.0 License
// http://creativecommons.org/licenses/by/3.0/

// NB: slow as molasses in FireFox, especially when sorting columns with a lot of text.
// An optimization is implemented which makes speed bearable, toggled by the following variable
var SORT_SPEED_OPT = true;
// a bit of browser preference: Opera does not need optimization
if(window.opera) { SORT_SPEED_OPT=false; }
// the optimization has one limitation on the functionality: when sorting search
// results, the expanded info, e.g. bibtex/review, is collapsed. In the non-optimized
// version they remain visible.


if (window.addEventListener) {
	window.addEventListener("load",initSortTable,false) }
else if (window.attachEvent) {
	window.attachEvent("onload", initSortTable); }

function initSortTable() {
var alltables = document.getElementsByTagName('table');
for(i=0;i<alltables.length;i++) {
	var currentTable = alltables[i];
	if(currentTable.className.indexOf('sortable') !=-1) {
		var thead = currentTable.getElementsByTagName('thead')[0];
		thead.title = 'Click on any column header to sort';
		for (var i=0;cell = thead.getElementsByTagName('th')[i];i++) {
			cell.onclick = function () { resortTable(this); };
			// make it possible to have a default sort column
			if(cell.className.indexOf('sort')!=-1) {
				resortTable(cell)
			}
		}
	}
}
}

var SORT_COLUMN_INDEX

function resortTable(td) {
	var column = td.cellIndex;
	var table = getParent(td,'TABLE');

	var allRows = table.getElementsByTagName('tbody')[0].getElementsByTagName('tr');
	var newRows = new Array();

	for (var i=0, k=0; i<allRows.length;i++) {

		var rowclass = allRows[i].className;

		if (rowclass.indexOf('entry') != -1) {
	       	newRows[k++] = allRows[i];
		}
		
		if (SORT_SPEED_OPT) {
		// remove highlight class
		allRows[i].className = rowclass.replace(/highlight/,'');
		// close information
		if(rowclass.indexOf('entry') == -1 && rowclass.indexOf('noshow') == -1) { allRows[i].className = rowclass + ' noshow';}
		} 
	}


	// If other sort functions are deemed necessary (e.g. for
	// dates and currencies) they can be added.
	var sortfn = ts_sort_firstchild_caseinsensitive;
	SORT_COLUMN_INDEX = column;
	newRows.sort(sortfn);

	// create a container for showing sort arrow
	var arrow =  td.getElementsByTagName('span')[0];
	if (!arrow) { var arrow = td.appendChild(document.createElement('span'));}
	
	if (td.className) {
		if (td.className.indexOf('sort_asc') !=-1) {
			td.className = td.className.replace(/_asc/,"_des");
			newRows.reverse();
			arrow.innerHTML = '&uArr;';
		} else if (td.className.indexOf('sort_des') !=-1) {
			td.className = td.className.replace(/_des/,"_asc");
			arrow.innerHTML = '&dArr;';
		} else { 
			td.className += ' sort_asc'; 
			arrow.innerHTML = '&dArr;';
		}
	} else {
		td.className += 'sort_asc';
		arrow.innerHTML = '&dArr;';
	}
	
	// Remove the classnames and up/down arrows for the other headers
	var ths = table.getElementsByTagName('thead')[0].getElementsByTagName('th');
	for (var i=0; i<ths.length; i++) {
		if(ths[i]!=td && ths[i].className.indexOf('sort_')!=-1) {
		// argh, moronic JabRef thinks (backslash)w is an output field!!
		//ths[i].className = ths[i].className.replace(/sort_(backslash)w{3}/,"");
		ths[i].className = ths[i].className.replace(/sort_asc/,"");
		ths[i].className = ths[i].className.replace(/sort_des/,"");

		// remove span
		var arrow =  ths[i].getElementsByTagName('span')[0];
		if (arrow) { ths[i].removeChild(arrow); }
		}
	}

	// We appendChild rows that already exist to the tbody, so it moves them rather than creating new ones
	for (i=0;i<newRows.length;i++) { 
		table.getElementsByTagName('tbody')[0].appendChild(newRows[i]);

		if(!SORT_SPEED_OPT){
		// moving additional information, e.g. bibtex/abstract to right locations
		// this allows to sort, even with abstract/review/etc. still open
		var articleid = newRows[i].id;

		var entry = document.getElementById(articleid);
		var abs = document.getElementById('abs_'+articleid);
		var rev = document.getElementById('rev_'+articleid);
		var bib = document.getElementById('bib_'+articleid);		
	
		var tbody = table.getElementsByTagName('tbody')[0];
		// mind the order of adding the entries
		if(abs) { tbody.appendChild(abs); }
		if(rev) { tbody.appendChild(rev); }
		if(bib) { tbody.appendChild(bib); }
		}
	}
}

function ts_sort_firstchild_caseinsensitive(a,b) {
	// only search in .firstChild of the cells. Speeds things up tremendously in FF
	// problem is that it won't find all the text in a cell if the firstChild is an element
	// or if there are other elements in the cell. Risky fix, but the speed boost is worth it.
	var acell = a.cells[SORT_COLUMN_INDEX];
	var bcell = b.cells[SORT_COLUMN_INDEX];
	
	acell.firstChild? aa = getTextContent(acell.firstChild).toLowerCase():aa = "";
	bcell.firstChild? bb = getTextContent(bcell.firstChild).toLowerCase():bb = "";

	if (aa==bb) return 0;
	if (aa<bb) return -1;
	return 1;
}

function ts_sort_caseinsensitive(a,b) {
	aa = getTextContent(a.cells[SORT_COLUMN_INDEX]).toLowerCase();
	bb = getTextContent(b.cells[SORT_COLUMN_INDEX]).toLowerCase();
	if (aa==bb) return 0;
	if (aa<bb) return -1;
	return 1;
}

function ts_sort_default(a,b) {
	aa = getTextContent(a.cells[SORT_COLUMN_INDEX]);
	bb = getTextContent(b.cells[SORT_COLUMN_INDEX]);
	if (aa==bb) return 0;
	if (aa<bb) return -1;
	return 1;
}

function getParent(el, pTagName) {
	if (el == null) { 
		return null;
	} else if (el.nodeType == 1 && el.tagName.toLowerCase() == pTagName.toLowerCase()) {
		return el;
	} else {
		return getParent(el.parentNode, pTagName);
	}
}
-->
</script>
<style type="text/css">
body { background-color: white; font-family: "Trebuchet MS", Arial, sans-serif; font-size: 12px; line-height: 1.2; padding: 1em; color: #2E2E2E; }

#qs { width: auto; border-style: solid; border-color: gray; border-width: 1px 1px 0px 1px; padding: 0.5em 0.5em; display:none; position:relative; }
#qs form { padding: 0px; margin: 0px; }
#qs form p { padding: 0px; margin: 0px; }

.invalidsearch { background-color: red; }

table { border: 1px gray solid; width: 100%; empty-cells: show; }
th, td { border: 1px gray solid; padding: 0.5em; vertical-align: top;  }
td { text-align: left; vertical-align: top; }
th { background-color: #EFEFEF; }

td a { color: navy; text-decoration: none; }
td a:hover  { text-decoration: underline; }

tr.noshow { display: none;}

tr.highlight td { background-color: #F1F1F1; border-top: 2px black solid; font-weight: bold; }
tr.abstract td, tr.review td, tr.bibtex td { background-color: #F1F1F1; border-bottom: 2px black solid; }
tr.nextshow td { border-bottom: 1px gray solid; }

tr.bibtex pre { width: 100%; overflow: auto;}

p.infolinks { margin: 0.5em 0em 0em 0em; padding: 0px; }

#qssettings { padding: 0.5em; position: absolute; top: 0.2em; right: 0.2em; border: 1px gray solid; background-color: white; display: none; }
#qssettings p { font-weight: bold; cursor: pointer; }
#qssettings ul { display: none; list-style-type: none; padding-left: 0; margin: 0; }
#qssettings.active ul { display: block; }

@media print {
	p.infolinks, #qssettings, #qs { display: none !important; }
	table { border-width: 0px; }
	tr { page-break-inside: avoid; }
	tr > * + * + * + * + * {display: none; }
	thead tr::before { content: "Reference"; border: 1px gray solid; padding: 0.5em; vertical-align: top; font-weight: bold; text-align: center; display: table-cell; background-color: #EFEFEF; }
	tr[id]::before { content: attr(id); display: table-cell; border: 1px gray solid; padding: 0.5em; vertical-align: top; font-style: italic; }
}

th.sort_asc, th.sort_des { border: 2px black solid; }
</style>
</head>
<body>

<div id="qs">
	<form action="">
	<p>QuickSearch: <input type="text" name="qsfield" id="qsfield" autocomplete="off" title="Allows plain text as well as RegExp searches" /><input type="button" onclick="clearQS()" value="clear" />&nbsp; Number of matching entries: <span id="stat">0</span>.</p>
	<div id="qssettings">
		<p onclick="toggleQSettingsDialog()">Search Settings</p>
		<ul></ul>
	</div>
	</form>
</div>
<table id="qstable" class="sortable" border="1">
<thead><tr><th width="20%">Author</th><th width="30%">Title</th><th width="5%">Year</th><th width="30%">Journal/Proceedings</th><th width="10%">Reftype</th><th width="5%">DOI/URL</th></tr></thead>
<tbody><tr id="albrecht02:lsa_machin" class="entry">
	<td>Albrecht, A., Lappas, G., Vinterbo, S.A., Wong, C. &amp; Ohno-Machado, L.</td>
	<td>Two Applications of the LSA Machine <p class="infolinks">[<a href="javascript:toggleInfo('albrecht02:lsa_machin','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Proceedings of the 9$^th$ International Conference on Neural Information Processing (ICONIP'02)&nbsp;</td>
	<td>inproceedings</td>
	<td><a href="http://dx.doi.org/10.1109/ICONIP.2002.1202156">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_albrecht02:lsa_machin" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inproceedings{albrecht02:lsa_machin,
  author = {A. Albrecht and G. Lappas and S. A. Vinterbo and C.K. Wong and L. Ohno-Machado},
  title = {Two Applications of the LSA Machine},
  booktitle = {Proceedings of the 9$^th$ International Conference on Neural Information Processing (ICONIP'02)},
  year = {2002},
  note = {ISBN 981-04-7525-X},
  doi = {http://dx.doi.org/10.1109/ICONIP.2002.1202156}
}
</pre></td>
</tr>
<tr id="albrecht02:_epicur" class="entry">
	<td>Albrecht, A., Vinterbo, S.A. &amp; Ohno-Machado, L.</td>
	<td>Epicurean-style Learning Applied to the Classification of Gene-Expression Data <p class="infolinks">[<a href="javascript:toggleInfo('albrecht02:_epicur','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Research and Development in Intelligent Systems XIX, pp. 47-59&nbsp;</td>
	<td>incollection</td>
	<td><a href="http://dx.doi.org/10.1007/978-1-4471-0651-7_4">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_albrecht02:_epicur" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@incollection{albrecht02:_epicur,
  author = {A. Albrecht and S. A. Vinterbo and L. Ohno-Machado},
  title = {Epicurean-style Learning Applied to the Classification of Gene-Expression Data},
  booktitle = {Research and Development in Intelligent Systems XIX},
  publisher = {Springer-Verlag},
  year = {2002},
  pages = {47-59},
  note = {ISBN 1-85233-674-9},
  doi = {http://dx.doi.org/10.1007/978-1-4471-0651-7_4}
}
</pre></td>
</tr>
<tr id="Albrecht2003" class="entry">
	<td>Albrecht, A., Vinterbo, S.A. &amp; Ohno-Machado, L.</td>
	<td>An Epicurean learning approach to gene-expression data classification. <p class="infolinks">[<a href="javascript:toggleInfo('Albrecht2003','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Albrecht2003','bibtex')">BibTeX</a>]</p></td>
	<td>2003</td>
	<td>Artif Intell Med<br/>Vol. 28(1), pp. 75-87&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0933-3657(03)00036-8">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Albrecht2003" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We investigate the use of perceptrons for classification of microarray data where we use two datasets that were published in [Nat. Med. 7 (6) (2001) 673] and [Science 286 (1999) 531]. The classification problem studied by Khan et al. is related to the diagnosis of small round blue cell tumours (SRBCT) of childhood which are difficult to classify both clinically and via routine histology. Golub et al. study acute myeloid leukemia (AML) and acute lymphoblastic leukemia (ALL). We used a simulated annealing-based method in learning a system of perceptrons, each obtained by resampling of the training set. Our results are comparable to those of Khan et al. and Golub et al., indicating that there is a role for perceptrons in the classification of tumours based on gene-expression data. We also show that it is critical to perform feature selection in this type of models, i.e. we propose a method for identifying genes that might be significant for the particular tumour types. For SRBCTs, zero error on test data has been obtained for only 13 out of 2308 genes; for the ALL/AML problem, we have zero error for 9 out of 7129 genes that are used for the classification procedure. Furthermore, we provide evidence that Epicurean-style learning and simulated annealing-based search are both essential for obtaining the best classification results.</td>
</tr>
<tr id="bib_Albrecht2003" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Albrecht2003,
  author = {Andreas Albrecht and Staal A Vinterbo and Lucila Ohno-Machado},
  title = {An Epicurean learning approach to gene-expression data classification.},
  journal = {Artif Intell Med},
  year = {2003},
  volume = {28},
  number = {1},
  pages = {75-87},
  doi = {http://dx.doi.org/10.1016/S0933-3657(03)00036-8}
}
</pre></td>
</tr>
<tr id="albrecht02:_simul_anneal" class="entry">
	<td>Albrecht, A., Vinterbo, S.A., Wong, C. &amp; Ohno-Machado, L.</td>
	<td>A Simulated Annealing and Resampling Method for Training Perceptrons to Classify Gene-Expression Data <p class="infolinks">[<a href="javascript:toggleInfo('albrecht02:_simul_anneal','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td><br/>Vol. 2415Artificial Neural Networks (ICANN'02), pp. 401-406&nbsp;</td>
	<td>incollection</td>
	<td><a href="http://dx.doi.org/10.1007/3-540-46084-5_65">DOI</a> <a href="self/Proceedings/ICANN.ps">URL</a>&nbsp;</td>
</tr>
<tr id="bib_albrecht02:_simul_anneal" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@incollection{albrecht02:_simul_anneal,
  author = {A. Albrecht and S. A. Vinterbo and C.K. Wong and L. Ohno-Machado},
  title = {A Simulated Annealing and Resampling Method for Training Perceptrons to Classify Gene-Expression Data},
  booktitle = {Artificial Neural Networks (ICANN'02)},
  publisher = {Springer-Verlag},
  year = {2002},
  volume = {2415},
  pages = {401-406},
  note = {ISBN 3-540-44074-7},
  url = {self/Proceedings/ICANN.ps},
  doi = {http://dx.doi.org/10.1007/3-540-46084-5_65}
}
</pre></td>
</tr>
<tr id="Chaudhuri2013" class="entry">
	<td>Chaudhuri, K. &amp; Vinterbo, S.A.</td>
	<td>A Stability-based Validation Procedure for Differentially Private Machine Learning <p class="infolinks">[<a href="javascript:toggleInfo('Chaudhuri2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>NIPS&nbsp;</td>
	<td>inproceedings</td>
	<td><a href="http://papers.nips.cc/paper/5014-a-stability-based-validation-procedure-for-differentially-private-machine-learning.pdf">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Chaudhuri2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inproceedings{Chaudhuri2013,
  author = {Kamalika Chaudhuri and Staal A Vinterbo},
  title = {A Stability-based Validation Procedure for Differentially Private Machine Learning},
  booktitle = {NIPS},
  year = {2013},
  url = {http://papers.nips.cc/paper/5014-a-stability-based-validation-procedure-for-differentially-private-machine-learning.pdf}
}
</pre></td>
</tr>
<tr id="curtis_using_2009" class="entry">
	<td>Curtis, D.W., Bailey, J., Pino, E.J., Stair, T., Vinterbo, S., Waterman, J., Shih, E.I., Guttag, J.V., Greenes, R.A. &amp; Ohno-Machado, L.</td>
	<td>Using ambient intelligence for physiological monitoring <p class="infolinks">[<a href="javascript:toggleInfo('curtis_using_2009','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('curtis_using_2009','bibtex')">BibTeX</a>]</p></td>
	<td>2009</td>
	<td>Journal of Ambient Intelligence and Smart Environments<br/>Vol. 1(2), pp. 129-142&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.3233/AIS-2009-0018">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_curtis_using_2009" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Ambient intelligence is a way of subtly gathering information from an environment and acting on it. In the field of physiological monitoring, there are several goals that ambient intelligence can help us achieve. First, when patients are anxious, unobtrusive monitoring does not aggravate their anxiety. Second, when patients are at risk and there are insufficient caregivers to attend to each patient individually in a timely manner, unobtrusive pervasive monitoring can reassure patients that they are being cared for. Furthermore, caregivers appreciate being able to monitor more patients. The SMART system was developed to monitor patients' vital signs and locations in the waiting area of a hospital's emergency department. This paper reviews the SMART system and compares it to several other systems in related areas.</td>
</tr>
<tr id="bib_curtis_using_2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{curtis_using_2009,
  author = {Dorothy W. Curtis and Jacob Bailey and Esteban J. Pino and Thomas Stair and Staal Vinterbo and Jason Waterman and Eugene I. Shih and John V. Guttag and Robert A. Greenes and Lucila Ohno-Machado},
  title = {Using ambient intelligence for physiological monitoring},
  journal = {Journal of Ambient Intelligence and Smart Environments},
  year = {2009},
  volume = {1},
  number = {2},
  pages = {129--142},
  doi = {http://dx.doi.org/10.3233/AIS-2009-0018}
}
</pre></td>
</tr>
<tr id="Curtis2008" class="entry">
	<td>Curtis, D.W., Pino, E.J., Bailey, J.M., Shih, E.I., Waterman, J., Vinterbo, S.A., Stair, T.O., Guttag, J.V., Greenes, R.A. &amp; Ohno-Machado, L.</td>
	<td>SMART -- An Integrated Wireless System for Monitoring Unattended Patients <p class="infolinks">[<a href="javascript:toggleInfo('Curtis2008','bibtex')">BibTeX</a>]</p></td>
	<td>2008</td>
	<td>JAMIA<br/>Vol. 15(1), pp. 44-53&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1197/jamia.M2016">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Curtis2008" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Curtis2008,
  author = {Dorothy W. Curtis and Esteban J. Pino and Jacob M. Bailey and Eugene I. Shih and Jason Waterman and Staal A. Vinterbo and Thomas O. Stair and John V. Guttag and Robert A. Greenes and Lucila Ohno-Machado},
  title = {SMART -- An Integrated Wireless System for Monitoring Unattended Patients},
  journal = {JAMIA},
  year = {2008},
  volume = {15},
  number = {1},
  pages = {44-53},
  doi = {http://dx.doi.org/10.1197/jamia.M2016}
}
</pre></td>
</tr>
<tr id="Dreiseitl2007" class="entry">
	<td>Dreiseitl, S., Binder, M., Vinterbo, S. &amp; Kittler, H.</td>
	<td>Applying a decision support system in clinical practice: Results from melanoma diagnosis <p class="infolinks">[<a href="javascript:toggleInfo('Dreiseitl2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dreiseitl2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>AMIA ProceedingsProceedings of AMIA 2007&nbsp;</td>
	<td>inproceedings</td>
	<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2655775/">URL</a>&nbsp;</td>
</tr>
<tr id="abs_Dreiseitl2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We investigate the performance of a decision-support tool for the diagnosis of pigmented skin lesions. Previously, we had developed andvalidated a neural network-based systems for automated melanoma diagnosis. The work reported in this paper investigates the use of this system in a real-world clinical trial with 511 patients and 3827 lesion evaluations. We analyzed a number of outcomes of the trial, such as direct comparison of system performance in laboratory and clinical setting, the performance of physicians using the system compared to a control dermatologist without the system, and repeatability of system recommendations.<br> The results show that system performance was significantly less inthe laboratory setting compared to the real-world setting. Dermatologists using the system, however, achieved higher sensitivities and specificities than numbers reported in the literature for physicians of similar training who did not use a decision-support system. We also show that the process of acquiring lesion images using digital dermoscopy devices needs to be standardized before sufficiently high repeatability of measurements can be assured.</td>
</tr>
<tr id="bib_Dreiseitl2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inproceedings{Dreiseitl2007,
  author = {Stephan Dreiseitl and Michael Binder and Staal Vinterbo and Harald Kittler},
  title = {Applying a decision support system in clinical practice: Results from melanoma diagnosis},
  booktitle = {Proceedings of AMIA 2007},
  journal = {AMIA Proceedings},
  year = {2007},
  url = {http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2655775/}
}
</pre></td>
</tr>
<tr id="Dreiseitl2001" class="entry">
	<td>Dreiseitl, S., Ohno-Machado, L., Kittler, H., Vinterbo, S., Billhardt, H. &amp; Binder, M.</td>
	<td>A comparison of machine learning methods for the diagnosis of pigmented skin lesions. <p class="infolinks">[<a href="javascript:toggleInfo('Dreiseitl2001','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Dreiseitl2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>J Biomed Inform<br/>Vol. 34(1), pp. 28-36&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1006/jbin.2001.1004">DOI</a> <a href="http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.13.2630">URL</a>&nbsp;</td>
</tr>
<tr id="abs_Dreiseitl2001" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: We analyze the discriminatory power of k-nearest neighbors, logistic regression, artificial neural networks (ANNs), decision tress, and support vector machines (SVMs) on the task of classifying pigmented skin lesions as common nevi, dysplastic nevi, or melanoma. Three different classification tasks were used as benchmarks: the dichotomous problem of distinguishing common nevi from dysplastic nevi and melanoma, the dichotomous problem of distinguishing melanoma from common and dysplastic nevi, and the trichotomous problem of correctly distinguishing all three classes. Using ROC analysis to measure the discriminatory power of the methods shows that excellent results for specific classification problems in the domain of pigmented skin lesions can be achieved with machine-learning methods. On both dichotomous and trichotomous tasks, logistic regression, ANNs, and SVMs performed on about the same level, with k-nearest neighbors and decision trees performing worse.</td>
</tr>
<tr id="bib_Dreiseitl2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dreiseitl2001,
  author = {S Dreiseitl and L Ohno-Machado and H Kittler and S Vinterbo and H Billhardt and M Binder},
  title = {A comparison of machine learning methods for the diagnosis of pigmented skin lesions.},
  journal = {J Biomed Inform},
  year = {2001},
  volume = {34},
  number = {1},
  pages = {28-36},
  url = {http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.13.2630},
  doi = {http://dx.doi.org/10.1006/jbin.2001.1004}
}
</pre></td>
</tr>
<tr id="Dreiseitl1999" class="entry">
	<td>Dreiseitl, S., Ohno-Machado, L. &amp; Vinterbo, S.</td>
	<td>Evaluating variable selection methods for diagnosis of myocardial infarction <p class="infolinks">[<a href="javascript:toggleInfo('Dreiseitl1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>JAMIA<br/>Vol. Suppl. S, pp. 246-250&nbsp;</td>
	<td>article</td>
	<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2232647/">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Dreiseitl1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dreiseitl1999,
  author = {Dreiseitl, S. and Ohno-Machado, L. and Vinterbo, S.},
  title = {Evaluating variable selection methods for diagnosis of myocardial infarction},
  journal = {JAMIA},
  year = {1999},
  volume = {Suppl. S},
  pages = {246--250},
  url = {http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2232647/}
}
</pre></td>
</tr>
<tr id="Dreiseitl2010" class="entry">
	<td>Dreiseitl, S., Osl, M., Baumgartner, C. &amp; Vinterbo, S.A.</td>
	<td>An evaluation of heuristics for rule ranking <p class="infolinks">[<a href="javascript:toggleInfo('Dreiseitl2010','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Artificial Intelligence in Medicine<br/>Vol. 50(3), pp. 175-180&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.artmed.2010.03.005">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Dreiseitl2010" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dreiseitl2010,
  author = {Stephan Dreiseitl and Melanie Osl and Christian Baumgartner and Staal A. Vinterbo},
  title = {An evaluation of heuristics for rule ranking},
  journal = {Artificial Intelligence in Medicine},
  year = {2010},
  volume = {50},
  number = {3},
  pages = {175-180},
  doi = {http://dx.doi.org/10.1016/j.artmed.2010.03.005}
}
</pre></td>
</tr>
<tr id="Dreiseitl2002" class="entry">
	<td>Dreiseitl, S., Vinterbo, S. &amp; Ohno-Machado, L.</td>
	<td>Disambiguation data: Extracting information from anonymized sources <p class="infolinks">[<a href="javascript:toggleInfo('Dreiseitl2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>JAMIA<br/>Vol. 9(90061), pp. S110-S114&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1197/jamia.M1240">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Dreiseitl2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Dreiseitl2002,
  author = {Dreiseitl, S. and Vinterbo, S. and Ohno-Machado, L.},
  title = {Disambiguation data: Extracting information from anonymized sources},
  journal = {JAMIA},
  year = {2002},
  volume = {9},
  number = {90061},
  pages = {S110--S114},
  doi = {http://dx.doi.org/10.1197/jamia.M1240}
}
</pre></td>
</tr>
<tr id="Goodwin2000" class="entry">
	<td>Goodwin, L.K., Maher, S.G., Ohno-Machado, L., Iannacchione, M.A., Crockett, P., Dreiseitl, S., Vinterbo, S. &amp; Hammond, W.E.</td>
	<td>Building knowledge in a complex preterm birth problem domain <p class="infolinks">[<a href="javascript:toggleInfo('Goodwin2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>JAMIA<br/>Vol. Suppl. S, pp. 305-309&nbsp;</td>
	<td>article</td>
	<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2243761/">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Goodwin2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Goodwin2000,
  author = {Goodwin, L. K. and Maher, S. G. and Ohno-Machado, L. and Iannacchione, M. A. and Crockett, P. and Dreiseitl, S. and Vinterbo, S. and Hammond, W. E.},
  title = {Building knowledge in a complex preterm birth problem domain},
  journal = {JAMIA},
  year = {2000},
  volume = {Suppl. S},
  pages = {305--309},
  url = {http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2243761/}
}
</pre></td>
</tr>
<tr id="Jenssen2000" class="entry">
	<td>Jenssen, T.K. &amp; Vinterbo, S.</td>
	<td>A set-covering approach to specific search for literature about human genes <p class="infolinks">[<a href="javascript:toggleInfo('Jenssen2000','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>JAMIA<br/>Vol. Suppl. S, pp. 384-388&nbsp;</td>
	<td>article</td>
	<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/11079910">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Jenssen2000" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Jenssen2000,
  author = {Jenssen, T. K. and Vinterbo, S.},
  title = {A set-covering approach to specific search for literature about human genes},
  journal = {JAMIA},
  year = {2000},
  volume = {Suppl. S},
  pages = {384--388},
  url = {http://www.ncbi.nlm.nih.gov/pubmed/11079910}
}
</pre></td>
</tr>
<tr id="vinterbo:genlit00" class="entry">
	<td>Jenssen, T.-K. &amp; Vinterbo, S.</td>
	<td>A Relational Approach to Defining Document Set Relevance: An Application in Human Genetics <p class="infolinks">[<a href="javascript:toggleInfo('vinterbo:genlit00','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>(7/00)&nbsp;</td>
	<td>techreport</td>
	<td><a href="self/techraps/scare-techrap.pdf">URL</a>&nbsp;</td>
</tr>
<tr id="bib_vinterbo:genlit00" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@techreport{vinterbo:genlit00,
  author = {T.-K. Jenssen and S. Vinterbo},
  title = {A Relational Approach to Defining Document Set Relevance: An Application in Human Genetics},
  year = {2000},
  number = {7/00},
  note = {ISSN 0802-6394},
  url = {self/techraps/scare-techrap.pdf}
}
</pre></td>
</tr>
<tr id="kim_anomaly_2011" class="entry">
	<td>Kim, J., Grillo, J.M., Boxwala, A.A., Jiang, X., Mandelbaum, R.B., Patel, B.A., Mikels, D., Vinterbo, S.A. &amp; Ohno-Machado, L.</td>
	<td>Anomaly and signature filtering improve classifier performance for detection of suspicious access to EHRs <p class="infolinks">[<a href="javascript:toggleInfo('kim_anomaly_2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('kim_anomaly_2011','bibtex')">BibTeX</a>]</p></td>
	<td>2011</td>
	<td>AMIA Annual Symposium Proceedings<br/>Vol. 2011, pp. 723-731&nbsp;</td>
	<td>article</td>
	<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3243249/">URL</a>&nbsp;</td>
</tr>
<tr id="abs_kim_anomaly_2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Our objective is to facilitate semi-automated detection of suspicious access to EHRs. Previously we have shown that a machine learning method can play a role in identifying potentially inappropriate access to EHRs. However, the problem of sampling informative instances to build a classifier still remained. We developed an integrated filtering method leveraging both anomaly detection based on symbolic clustering and signature detection, a rule-based technique. We applied the integrated filtering to 25.5 million access records in an intervention arm, and compared this with 8.6 million access records in a control arm where no filtering was applied. On the training set with cross-validation, the AUC was 0.960 in the control arm and 0.998 in the intervention arm. The difference in false negative rates on the independent test set was significant, P=1.6Ã10(-6). Our study suggests that utilization of integrated filtering strategies to facilitate the construction of classifiers can be helpful.</td>
</tr>
<tr id="bib_kim_anomaly_2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{kim_anomaly_2011,
  author = {Kim, Jihoon and Grillo, Janice M and Boxwala, Aziz A and Jiang, Xiaoqian and Mandelbaum, Rose B and Patel, Bhakti A and Mikels, Debra and Vinterbo, Staal A and Ohno-Machado, Lucila},
  title = {Anomaly and signature filtering improve classifier performance for detection of suspicious access to EHRs},
  journal = {AMIA Annual Symposium Proceedings},
  year = {2011},
  volume = {2011},
  pages = {723--731},
  note = {PMID: 22195129},
  url = {http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3243249/}
}
</pre></td>
</tr>
<tr id="Kuo2004" class="entry">
	<td>Kuo, W.P., Kim, E.-Y., Trimarchi, J., Jenssen, T.-K., Vinterbo, S.A. &amp; Ohno-Machado, L.</td>
	<td>A primer on gene expression and microarrays for machine learning researchers. <p class="infolinks">[<a href="javascript:toggleInfo('Kuo2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Kuo2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>J Biomed Inform<br/>Vol. 37(4), pp. 293-303&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jbi.2004.07.002">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Kuo2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Data originating from biomedical experiments has provided machine learning researchers with an important source of motivation for developing and evaluating new algorithms. A new wave of algorithmic development has been initiated with the publication of gene expression data derived from microarrays. Microarray data analysis is particularly challenging given the large number of measurements (typically in the order of thousands) that are reported for relatively few samples (typically in the order of dozens). Many data sets are now available on the web. It is important that machine learning researchers understand how data are obtained and which assumptions are necessary in the analysis. Microarray data have the potential to cause significant impact in machine learning research, not just as a rich and realistic source of cases for testing new algorithms, as has been the UCI machine learning repository in the past decades, but also as a main motivation for their development. In this article, we briefly review the biology underlying microarrays, the process of obtaining gene expression measurements, and the rationale behind the common types of analyses involved in a microarray experiment. We outline the main challenges and reiterate critical considerations regarding the construction of supervised learning models that use this type of data. The goal of this article is to familiarize machine learning researchers with data originated from gene expression microarrays.</td>
</tr>
<tr id="bib_Kuo2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Kuo2004,
  author = {Winston Patrick Kuo and Eun-Young Kim and Jeff Trimarchi and Tor-Kristian Jenssen and Staal A Vinterbo and Lucila Ohno-Machado},
  title = {A primer on gene expression and microarrays for machine learning researchers.},
  journal = {J Biomed Inform},
  year = {2004},
  volume = {37},
  number = {4},
  pages = {293-303},
  doi = {http://dx.doi.org/10.1016/j.jbi.2004.07.002}
}
</pre></td>
</tr>
<tr id="Lasko2009" class="entry">
	<td>Lasko, T.A. &amp; Vinterbo, S.A.</td>
	<td>Spectral Anonymization of Data <p class="infolinks">[<a href="javascript:toggleInfo('Lasko2009','bibtex')">BibTeX</a>]</p></td>
	<td>2010</td>
	<td>Knowledge and Data Engineering, IEEE Transactions on<br/>Vol. 22(3), pp. 437 -446&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1109/TKDE.2009.88">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Lasko2009" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lasko2009,
  author = {Thomas A. Lasko and Staal A. Vinterbo},
  title = {Spectral Anonymization of Data},
  journal = {Knowledge and Data Engineering, IEEE Transactions on},
  publisher = {IEEE Computer Society},
  year = {2010},
  volume = {22},
  number = {3},
  pages = {437 -446},
  doi = {http://dx.doi.org/10.1109/TKDE.2009.88}
}
</pre></td>
</tr>
<tr id="Lyumkis2014" class="entry">
	<td>Lyumkis, D., dos Passos, D.O., Tahara, E., Webb, K., Bennett, E.J., Vinterbo, S., Potter, C.S., Carragher, B. &amp; Joazeiro, C.A.</td>
	<td>Structural Basis for Translational Surveillance by the Large Ribosomal Subunit-Associated Protein Quality Control Complex <p class="infolinks">[<a href="javascript:toggleInfo('Lyumkis2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>PNAS&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1073/pnas.1413882111">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Lyumkis2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lyumkis2014,
  author = {Dmitry Lyumkis and Dario Oliveira dos Passos and Erich Tahara and Kristofor Webb and Eric J Bennett and Staal Vinterbo and Clinton S Potter and Bridget Carragher and Claudio A.P. Joazeiro},
  title = {Structural Basis for Translational Surveillance by the Large Ribosomal Subunit-Associated Protein Quality Control Complex},
  journal = {PNAS},
  year = {2014},
  note = {In Press},
  doi = {http://dx.doi.org/10.1073/pnas.1413882111}
}
</pre></td>
</tr>
<tr id="Lyumkis2013" class="entry">
	<td>Lyumkis, D., Vinterbo, S.A., Potter, C.S. &amp; Carragher, B.</td>
	<td>Optimod - An Automated Approach for Constructing and Optimizing Initial Models for Single-Particle Electron Microscopy <p class="infolinks">[<a href="javascript:toggleInfo('Lyumkis2013','bibtex')">BibTeX</a>]</p></td>
	<td>2013</td>
	<td>Journal of Structural Biology<br/>Vol. 184(3), pp. 417-426&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.jsb.2013.10.009">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Lyumkis2013" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Lyumkis2013,
  author = {Dmitry Lyumkis and Staal A. Vinterbo and Clinton S. Potter and Bridget Carragher},
  title = {Optimod - An Automated Approach for Constructing and Optimizing Initial Models for Single-Particle Electron Microscopy},
  journal = {Journal of Structural Biology},
  year = {2013},
  volume = {184},
  number = {3},
  pages = {417--426},
  doi = {http://dx.doi.org/10.1016/j.jsb.2013.10.009}
}
</pre></td>
</tr>
<tr id="Mehta2014" class="entry">
	<td>Mehta, S.R., Vinterbo, S.A. &amp; Little, S.J.</td>
	<td>Ensuring Privacy in the Study of Pathogen Genetics in the 21st Century <p class="infolinks">[<a href="javascript:toggleInfo('Mehta2014','bibtex')">BibTeX</a>]</p></td>
	<td>2014</td>
	<td>The Lancet Infectious Diseases<br/>Vol. In press.&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S1473-3099(14)70016-7">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Mehta2014" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Mehta2014,
  author = {Sanjay R. Mehta and Staal A. Vinterbo and Susan J. Little},
  title = {Ensuring Privacy in the Study of Pathogen Genetics in the 21st Century},
  journal = {The Lancet Infectious Diseases},
  year = {2014},
  volume = {In press.},
  doi = {http://dx.doi.org/10.1016/S1473-3099(14)70016-7}
}
</pre></td>
</tr>
<tr id="ohno-machado_idash:_2011" class="entry">
	<td>Ohno-Machado, L., Bafna, V., Boxwala, A.A., Chapman, B.E., Chapman, W.W., Chaudhuri, K., Day, M.E., Farcas, C., Heintzman, N.D., Jiang, X., Kim, H., Kim, J., Matheny, M.E., Resnic, F.S. &amp; Vinterbo, S.A.</td>
	<td>iDASH: integrating data for analysis, anonymization, and sharing <p class="infolinks">[<a href="javascript:toggleInfo('ohno-machado_idash:_2011','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('ohno-machado_idash:_2011','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>J Am Med Inform Assoc<br/>Vol. 19(2), pp. 196-201&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1136/amiajnl-2011-000538">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_ohno-machado_idash:_2011" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: iDASH (integrating data for analysis, anonymization, and sharing) is the newest National Center for Biomedical Computing funded by the NIH. It focuses on algorithms and tools for sharing data in a privacy-preserving manner. Foundational privacy technology research performed within iDASH is coupled with innovative engineering for collaborative tool development and data-sharing capabilities in a private Health Insurance Portability and Accountability Act (HIPAA)-certified cloud. Driving Biological Projects, which span different biological levels (from molecules to individuals to populations) and focus on various health conditions, help guide research and development within this Center. Furthermore, training and dissemination efforts connect the Center with its stakeholders and educate data owners and data consumers on how to share and use clinical and biological data. Through these various mechanisms, iDASH implements its goal of providing biomedical and behavioral researchers with access to data, software, and a high-performance computing environment, thus enabling them to generate and test new hypotheses.</td>
</tr>
<tr id="bib_ohno-machado_idash:_2011" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{ohno-machado_idash:_2011,
  author = {Lucila Ohno-Machado and Vineet Bafna and Aziz A. Boxwala and Brian E. Chapman and Wendy Webber Chapman and Kamalika Chaudhuri and Michele E. Day and Claudiu Farcas and Nathaniel D. Heintzman and Xiaoqian Jiang and Hyeoneui Kim and Jihoon Kim and Michael E. Matheny and Frederic S. Resnic and Staal A. Vinterbo},
  title = {iDASH: integrating data for analysis, anonymization, and sharing},
  journal = {J Am Med Inform Assoc},
  year = {2012},
  volume = {19},
  number = {2},
  pages = {196-201},
  note = {PMID: 22081224},
  doi = {http://dx.doi.org/10.1136/amiajnl-2011-000538}
}
</pre></td>
</tr>
<tr id="Ohno-Machado2004" class="entry">
	<td>Ohno-Machado, L., Silveira, P.S.P. &amp; Vinterbo, S.</td>
	<td>Protecting patient privacy by quantifiable control of disclosures in disseminated databases. <p class="infolinks">[<a href="javascript:toggleInfo('Ohno-Machado2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ohno-Machado2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Int J Med Inform<br/>Vol. 73(7-8), pp. 599-606&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.ijmedinf.2004.05.002">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Ohno-Machado2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: One of the fundamental rights of patients is to have their privacy protected by health care organizations, so that information that can be used to identify a particular individual is not used to reveal sensitive patient data such as diagnoses, reasons for ordering tests, test results, etc. A common practice is to remove sensitive data from databases that are disseminated to the public, but this can make the disseminated database useless for important public health purposes. If the degree of anonymity of a disseminated data set could be measured, it would be possible to design algorithms that can assure that the desired level of confidentiality is achieved. Privacy protection in disseminated databases can be facilitated by the use of special ambiguation algorithms. Most of these algorithms are aimed at making one individual indistinguishable from one or more of his peers. However, even in databases considered "anonymous", it may still be possible to obtain sensitive information about some individuals or groups of individuals with the use of pattern recognition algorithms. In this article, we study the problem of determining the degree of ambiguation in disseminated databases and discuss its implications in the development and testing of "anonymization" algorithms.</td>
</tr>
<tr id="bib_Ohno-Machado2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ohno-Machado2004,
  author = {Lucila Ohno-Machado and Paulo Sergio Panse Silveira and Staal Vinterbo},
  title = {Protecting patient privacy by quantifiable control of disclosures in disseminated databases.},
  journal = {Int J Med Inform},
  year = {2004},
  volume = {73},
  number = {7-8},
  pages = {599-606},
  doi = {http://dx.doi.org/10.1016/j.ijmedinf.2004.05.002}
}
</pre></td>
</tr>
<tr id="Ohno-Machado2001a" class="entry">
	<td>Ohno-Machado, L. &amp; Vinterbo, S.</td>
	<td>Effects of case removal in prognostic models. <p class="infolinks">[<a href="javascript:toggleInfo('Ohno-Machado2001a','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ohno-Machado2001a','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>Methods Inf Med<br/>Vol. 40(1), pp. 32-8&nbsp;</td>
	<td>article</td>
	<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/11310157">URL</a>&nbsp;</td>
</tr>
<tr id="abs_Ohno-Machado2001a" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Constructing and updating prognostic models that learn from training cases is a time-consuming task. The more compact, and yet informative, the training sets are, the faster one can build and properly evaluate such models. We have compared different regression diagnostic methods for selection and removal of training cases in prognostic models. Univariate determinations were performed using classical regression diagnostic statistics. Multivariate determinations were performed using (1) a sequential "backward" selection of cases, and (2) a non-sequential genetic algorithm. The genetic algorithm produced final models that kept few cases and retained predictive capability. A genetic algorithm approach to case selection may be better suited for guiding removal of cases in training sets than a univariate or a sequential multivariate approach, possibly because of its ability to detect sets of cases that are influential en bloc but may not be sufficiently influential when considered in isolation.</td>
</tr>
<tr id="bib_Ohno-Machado2001a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ohno-Machado2001a,
  author = {L Ohno-Machado and S Vinterbo},
  title = {Effects of case removal in prognostic models.},
  journal = {Methods Inf Med},
  year = {2001},
  volume = {40},
  number = {1},
  pages = {32-8},
  url = {http://www.ncbi.nlm.nih.gov/pubmed/11310157}
}
</pre></td>
</tr>
<tr id="lom:casesel99" class="entry">
	<td>Ohno-Machado, L. &amp; Vinterbo, S.</td>
	<td>Influential case detection in medical prognosis <p class="infolinks">[<a href="javascript:toggleInfo('lom:casesel99','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>AIMDM'99 Workshop on Prognostic Models in Medicine, pp. 33-37&nbsp;</td>
	<td>inproceedings</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_lom:casesel99" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inproceedings{lom:casesel99,
  author = {L. Ohno-Machado and S. Vinterbo},
  title = {Influential case detection in medical prognosis},
  booktitle = {AIMDM'99 Workshop on Prognostic Models in Medicine},
  year = {1999},
  pages = {33-37}
}
</pre></td>
</tr>
<tr id="Ohno-Machado2007" class="entry">
	<td>Ohno-Machado, L. &amp; Vinterbo, S.A.</td>
	<td>Data Mining in Biomedicine <p class="infolinks">[<a href="javascript:toggleInfo('Ohno-Machado2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td><br/>Vol. 1, pp. 305-319&nbsp;</td>
	<td>inbook</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Ohno-Machado2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inbook{Ohno-Machado2007,
  author = {Lucial Ohno-Machado and Staal A. Vinterbo},
  title = {Data Mining in Biomedicine},
  publisher = {World Scientific},
  year = {2007},
  volume = {1},
  pages = {305-319}
}
</pre></td>
</tr>
<tr id="ohno-machado02:fuzzy_gene" class="entry">
	<td>Ohno-Machado, L., Vinterbo, S.A. &amp; Weber, G.</td>
	<td>Classification of Gene Expression Data Using Fuzzy Logic <p class="infolinks">[<a href="javascript:toggleInfo('ohno-machado02:fuzzy_gene','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>Journal of Intelligent and Fuzzy Systems<br/>Vol. 12(1), pp. 19-24&nbsp;</td>
	<td>article</td>
	<td><a href="http://www.academia.edu/322135/Classification_of_Gene_Expression_Data_Using_Fuzzy_Logic">URL</a>&nbsp;</td>
</tr>
<tr id="bib_ohno-machado02:fuzzy_gene" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{ohno-machado02:fuzzy_gene,
  author = {L. Ohno-Machado and S. A. Vinterbo and G. Weber},
  title = {Classification of Gene Expression Data Using Fuzzy Logic},
  journal = {Journal of Intelligent and Fuzzy Systems},
  year = {2002},
  volume = {12},
  number = {1},
  pages = {19-24},
  url = {http://www.academia.edu/322135/Classification_of_Gene_Expression_Data_Using_Fuzzy_Logic}
}
</pre></td>
</tr>
<tr id="Ohno-Machado2001" class="entry">
	<td>Ohno-Machado, L., Vinterbo, S. &amp; Dreiseitl, S.</td>
	<td>Effects of data anonymization by cell suppression on descriptive statistics and predictive modeling performance <p class="infolinks">[<a href="javascript:toggleInfo('Ohno-Machado2001','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>JAMIA<br/>Vol. 9(90061), pp. S115-S119&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Ohno-Machado2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ohno-Machado2001,
  author = {Ohno-Machado, L. and Vinterbo, S. and Dreiseitl, S.},
  title = {Effects of data anonymization by cell suppression on descriptive statistics and predictive modeling performance},
  journal = {JAMIA},
  year = {2002},
  volume = {9},
  number = {90061},
  pages = {S115-S119}
}
</pre></td>
</tr>
<tr id="Ohno-Machado2002" class="entry">
	<td>Ohno-Machado, L., Vinterbo, S., Dreiseitl, S., Jenssen, T.-K. &amp; Kuo, W.</td>
	<td>Comparing imperfect measurements with the Bland-Altman technique: application in gene expression analysis. <p class="infolinks">[<a href="javascript:toggleInfo('Ohno-Machado2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Ohno-Machado2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>JAMIA<br/>Vol. Suppl. S, pp. 572-6&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Ohno-Machado2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Several problems in medicine and biology involve the comparison of two measurements made on the same set of cases. The problem differs from a calibration problem because no gold standard can be identified. Testing the null hypothesis of no relationship using measures of association is not optimal since the measurements are made on the same cases, and therefore correlation coefficients will tend to be significant. The descriptive Bland-Altman method can be used in exploratory analysis of this problem, allowing the visualization of gross systematic differences between the two sets of measurements. We utilize the method on three sets of matched observations and demonstrate its usefulness in detecting systematic variations between two measurement technologies to assess gene expression.</td>
</tr>
<tr id="bib_Ohno-Machado2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ohno-Machado2002,
  author = {Lucila Ohno-Machado and Staal Vinterbo and Stephen Dreiseitl and Tor-Kristian Jenssen and Winston Kuo},
  title = {Comparing imperfect measurements with the Bland-Altman technique: application in gene expression analysis.},
  journal = {JAMIA},
  year = {2002},
  volume = {Suppl. S},
  pages = {572-6}
}
</pre></td>
</tr>
<tr id="Ohno-Machado1999" class="entry">
	<td>Ohno-Machado, L., Vinterbo, S., Ohrn, A. &amp; Dreiseitl, S.</td>
	<td>Clinical data processing tools: A machine learning resource <p class="infolinks">[<a href="javascript:toggleInfo('Ohno-Machado1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>JAMIA<br/>Vol. Suppl. SProc AMIA Symp, pp. 1132-1132&nbsp;</td>
	<td>inproceedings</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Ohno-Machado1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inproceedings{Ohno-Machado1999,
  author = {Ohno-Machado, L. and Vinterbo, S. and Ohrn, A. and Dreiseitl, S.},
  title = {Clinical data processing tools: A machine learning resource},
  booktitle = {Proc AMIA Symp},
  journal = {JAMIA},
  year = {1999},
  volume = {Suppl. S},
  pages = {1132--1132},
  note = {Symposium abstract}
}
</pre></td>
</tr>
<tr id="Ohrn1997" class="entry">
	<td>Ohrn, A., Vinterbo, S., Szymanski, P. &amp; Komorowski, J.</td>
	<td>Modelling cardiac patient set residuals using rough sets <p class="infolinks">[<a href="javascript:toggleInfo('Ohrn1997','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td>JAMIA<br/>Vol. Suppl. S, pp. 203-207&nbsp;</td>
	<td>article</td>
	<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/9357617">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Ohrn1997" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Ohrn1997,
  author = {Ohrn, A. and Vinterbo, S. and Szymanski, P. and Komorowski, J.},
  title = {Modelling cardiac patient set residuals using rough sets},
  journal = {JAMIA},
  year = {1997},
  volume = {Suppl. S},
  pages = {203--207},
  url = {http://www.ncbi.nlm.nih.gov/pubmed/9357617}
}
</pre></td>
</tr>
<tr id="vinterbo99:phd" class="entry">
	<td>Vinterbo, S.</td>
	<td>Predictive Models in Medicine: Some Methods for Construction and Adaptation <p class="infolinks">[<a href="javascript:toggleInfo('vinterbo99:phd','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td><i>School</i>: Norwegian University of Science and Technology&nbsp;</td>
	<td>phdthesis</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_vinterbo99:phd" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@phdthesis{vinterbo99:phd,
  author = {S. Vinterbo},
  title = {Predictive Models in Medicine: Some Methods for Construction and Adaptation},
  school = {Norwegian University of Science and Technology},
  year = {1999},
  note = {ISBN 82-7984-011-7, ISSN 0802-6394}
}
</pre></td>
</tr>
<tr id="sv:master94" class="entry">
	<td>Vinterbo, S.</td>
	<td>Infinite Families of Simple Groups <p class="infolinks">[<a href="javascript:toggleInfo('sv:master94','bibtex')">BibTeX</a>]</p></td>
	<td>1994</td>
	<td><i>School</i>: Norwegian Institute of Technology (NTH)&nbsp;</td>
	<td>mastersthesis</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_sv:master94" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@mastersthesis{sv:master94,
  author = {S. Vinterbo},
  title = {Infinite Families of Simple Groups},
  school = {Norwegian Institute of Technology (NTH)},
  year = {1994}
}
</pre></td>
</tr>
<tr id="Vinterbo2012a" class="entry">
	<td>Vinterbo, S.A.</td>
	<td>Differentially Private Projected Histograms: Construction and Use for Prediction <p class="infolinks">[<a href="javascript:toggleInfo('Vinterbo2012a','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td><br/>Vol. 7524Proceedings of ECML-PKDD 2012, September 24-28, Bristol UK., pp. 19-34&nbsp;</td>
	<td>inproceedings</td>
	<td><a href="http://dx.doi.org/10.1007/978-3-642-33486-3_2">DOI</a> <a href="http://www.cs.bris.ac.uk/~flach/ECMLPKDD2012papers/1125758.pdf">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Vinterbo2012a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inproceedings{Vinterbo2012a,
  author = {Staal A. Vinterbo},
  title = {Differentially Private Projected Histograms: Construction and Use for Prediction},
  booktitle = {Proceedings of ECML-PKDD 2012, September 24-28, Bristol UK.},
  publisher = {Springer Verlag},
  year = {2012},
  volume = {7524},
  pages = {19-34},
  url = {http://www.cs.bris.ac.uk/~flach/ECMLPKDD2012papers/1125758.pdf},
  doi = {http://dx.doi.org/10.1007/978-3-642-33486-3_2}
}
</pre></td>
</tr>
<tr id="Vinterbo2007" class="entry">
	<td>Vinterbo, S.A.</td>
	<td>A Stab at Approximating Minimum Subadditive Join <p class="infolinks">[<a href="javascript:toggleInfo('Vinterbo2007','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Vinterbo2007','bibtex')">BibTeX</a>]</p></td>
	<td>2007</td>
	<td>Algorithms and Data Structures: 10th International Workshop, WADS 2007, Halifax, Canada, August 15-17, 2007 Proceedings&nbsp;</td>
	<td>inproceedings</td>
	<td><a href="http://dx.doi.org/10.1007/978-3-540-73951-7_19">DOI</a> <a href="self/Proceedings/minicover-wads2007.pdf">URL</a>&nbsp;</td>
</tr>
<tr id="abs_Vinterbo2007" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Let (L, ∗) be a semilattice, and let c:L -> [0, ∞) be mono-<br>tone and increasing on L. We state the Minimum Join problem as: given<br>size n sub-collection X of L and integer k with 1 <= k <= n, find a size k<br>sub-collection (x1 , x2 , . . . , xk ) of X that minimizes <br>c(x1 ∗ x2 ∗ · · · ∗ xk ).<br>If c(a ∗ b) <= c(a) + c(b) holds, we call this the Minimum Subadditive<br>Join (MSJ) problem and present a greedy (k − p + 1)-approximation al-<br>gorithm requiring O((k − p)n + n^p) joins for constant integer 0 < p <= k.<br>We show that the MSJ Minimum Coverage problem of selecting k out<br>of n finite sets such that their union is minimal is essentially as hard to<br>approximate as the Maximum Balanced Complete Bipartite Subgraph<br>(MBCBS) problem. The motivating by-product of the above is that the<br>privacy in databases related k-ambiguity problem over L with subaddi-<br>tive information loss can be approximated within k − p, and that the<br>k-ambiguity problem is essentially at least as hard to approximate as<br>MBCBS.</td>
</tr>
<tr id="bib_Vinterbo2007" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inproceedings{Vinterbo2007,
  author = {Staal A. Vinterbo},
  title = {A Stab at Approximating Minimum Subadditive Join},
  booktitle = {Algorithms and Data Structures: 10th International Workshop, WADS 2007, Halifax, Canada, August 15-17, 2007 Proceedings},
  publisher = {Springer Verlag},
  year = {2007},
  url = {self/Proceedings/minicover-wads2007.pdf},
  doi = {http://dx.doi.org/10.1007/978-3-540-73951-7_19}
}
</pre></td>
</tr>
<tr id="vinterbo03:_privac" class="entry">
	<td>Vinterbo, S.A.</td>
	<td>Privacy: A Machine Learning View <p class="infolinks">[<a href="javascript:toggleInfo('vinterbo03:_privac','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>IEEE Transactions on Knowledge and Data Engineering<br/>Vol. 16(8), pp. 939-948&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1109/TKDE.2004.31">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_vinterbo03:_privac" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{vinterbo03:_privac,
  author = {Staal A. Vinterbo},
  title = {Privacy: A Machine Learning View},
  journal = {IEEE Transactions on Knowledge and Data Engineering},
  year = {2004},
  volume = {16},
  number = {8},
  pages = {939-948},
  doi = {http://dx.doi.org/10.1109/TKDE.2004.31}
}
</pre></td>
</tr>
<tr id="vinterbo04:fastMDA" class="entry">
	<td>Vinterbo, S.A.</td>
	<td>A fast MDA approximation algorithm <p class="infolinks">[<a href="javascript:toggleInfo('vinterbo04:fastMDA','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>(DSG-TR-2004-04)&nbsp;</td>
	<td>techreport</td>
	<td><a href="self/techraps/fastMDA-techrap.pdf">URL</a>&nbsp;</td>
</tr>
<tr id="bib_vinterbo04:fastMDA" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@techreport{vinterbo04:fastMDA,
  author = {Staal A. Vinterbo},
  title = {A fast MDA approximation algorithm},
  year = {2004},
  number = {DSG-TR-2004-04},
  url = {self/techraps/fastMDA-techrap.pdf}
}
</pre></td>
</tr>
<tr id="vinterbo02:_maxim_np" class="entry">
	<td>Vinterbo, S.A.</td>
	<td>Maximum $k$-intersection, edge labeled multigraph max capacity $k$-path, and max factor $k$-gcd are all NP-hard <p class="infolinks">[<a href="javascript:toggleInfo('vinterbo02:_maxim_np','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>(DSG-TR-2002-12)&nbsp;</td>
	<td>techreport</td>
	<td><a href="self/techraps/kgcdmaxfactor.pdf">URL</a>&nbsp;</td>
</tr>
<tr id="bib_vinterbo02:_maxim_np" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@techreport{vinterbo02:_maxim_np,
  author = {Staal A. Vinterbo},
  title = {Maximum $k$-intersection, edge labeled multigraph max capacity $k$-path, and max factor $k$-gcd are all NP-hard},
  year = {2002},
  number = {DSG-TR-2002-12},
  url = {self/techraps/kgcdmaxfactor.pdf}
}
</pre></td>
</tr>
<tr id="vinterbo02:_note_hardn_ambig_probl" class="entry">
	<td>Vinterbo, S.A.</td>
	<td>A Note on the Hardness of the $k$-Ambiguity Problem <p class="infolinks">[<a href="javascript:toggleInfo('vinterbo02:_note_hardn_ambig_probl','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>(DSG-TR-2002-006)&nbsp;</td>
	<td>techreport</td>
	<td><a href="self/techraps/ambighardness.pdf">URL</a>&nbsp;</td>
</tr>
<tr id="bib_vinterbo02:_note_hardn_ambig_probl" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@techreport{vinterbo02:_note_hardn_ambig_probl,
  author = {Staal A. Vinterbo},
  title = {A Note on the Hardness of the $k$-Ambiguity Problem},
  year = {2002},
  number = {DSG-TR-2002-006},
  url = {self/techraps/ambighardness.pdf}
}
</pre></td>
</tr>
<tr id="Vinterbo2006a" class="entry">
	<td>Vinterbo, S.A. &amp; Dreiseitl, S.</td>
	<td>A Note on Solution Sizes in The Haplotype Tagging SNPS Problem <p class="infolinks">[<a href="javascript:toggleInfo('Vinterbo2006a','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>Proceedings of the 3rd European Modelling and Simulation Symposium (EMSS2006)&nbsp;</td>
	<td>inproceedings</td>
	<td><a href="self/Proceedings/emss2006.pdf">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Vinterbo2006a" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inproceedings{Vinterbo2006a,
  author = {Staal A. Vinterbo and Stephan Dreiseitl},
  title = {A Note on Solution Sizes in The Haplotype Tagging SNPS Problem},
  booktitle = {Proceedings of the 3rd European Modelling and Simulation Symposium (EMSS2006)},
  year = {2006},
  url = {self/Proceedings/emss2006.pdf}
}
</pre></td>
</tr>
<tr id="Vinterbo2006" class="entry">
	<td>Vinterbo, S.A., Dreiseitl, S. &amp; Ohno-Machado, L.</td>
	<td>Approximation properties of haplotype tagging. <p class="infolinks">[<a href="javascript:toggleInfo('Vinterbo2006','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Vinterbo2006','bibtex')">BibTeX</a>]</p></td>
	<td>2006</td>
	<td>BMC Bioinformatics<br/>Vol. 7, pp. 8&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1186/1471-2105-7-8">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Vinterbo2006" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: BACKGROUND: Single nucleotide polymorphisms (SNPs) are locations at which the genomic sequences of population members differ. Since these differences are known to follow patterns, disease association studies are facilitated by identifying SNPs that allow the unique identification of such patterns. This process, known as haplotype tagging, is formulated as a combinatorial optimization problem and analyzed in terms of complexity and approximation properties. RESULTS: It is shown that the tagging problem is NP-hard but approximable within 1 + ln((n2 - n)/2) for n haplotypes but not approximable within (1-epsilon) ln(n/2) for any epsilon > 0 unless NP subset DTIME(n(log log n)). A simple, very easily implementable algorithm that exhibits the above upper bound on solution quality is presented. This algorithm has running time O(np/2(2m-p+1)) < or = O(m(n2-n)/2) where p < or = min(n, m) for n haplotypes of size m. As we show that the approximation bound is asymptotically tight, the algorithm presented is optimal with respect to this asymptotic bound. CONCLUSION: The haplotype tagging problem is hard, but approachable with a fast, practical, and surprisingly simple algorithm that cannot be significantly improved upon on a single processor machine. Hence, significant improvement in computational efforts expended can only be expected if the computational effort is distributed and done in parallel.</td>
</tr>
<tr id="bib_Vinterbo2006" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Vinterbo2006,
  author = {Staal A Vinterbo and Stephan Dreiseitl and Lucila Ohno-Machado},
  title = {Approximation properties of haplotype tagging.},
  journal = {BMC Bioinformatics},
  year = {2006},
  volume = {7},
  pages = {8},
  doi = {http://dx.doi.org/10.1186/1471-2105-7-8}
}
</pre></td>
</tr>
<tr id="Vinterbo2005" class="entry">
	<td>Vinterbo, S.A., Kim, E.-Y. &amp; Ohno-Machado, L.</td>
	<td>Small, fuzzy and interpretable gene expression based classifiers. <p class="infolinks">[<a href="javascript:toggleInfo('Vinterbo2005','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Vinterbo2005','bibtex')">BibTeX</a>]</p></td>
	<td>2005</td>
	<td>Bioinformatics<br/>Vol. 21(9), pp. 1964-1970&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1093/bioinformatics/bti287">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Vinterbo2005" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: MOTIVATION: Interpretation of classification models derived from gene expression data is usually not simple, yet it is an important aspect in the analytical process. We investigate the performance of small rule-based classifiers based on fuzzy logic in five data sets that are different in size, laboratory origin, and biomedical domain. RESULTS: The classifiers resulted in rules that can be readily examined by biomedical researchers. The fuzzy-logic-based classifiers compare favorably with logistic regression in all data sets. AVAILABILITY: Prototype available upon request.</td>
</tr>
<tr id="bib_Vinterbo2005" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Vinterbo2005,
  author = {Staal A Vinterbo and Eun-Young Kim and Lucila Ohno-Machado},
  title = {Small, fuzzy and interpretable gene expression based classifiers.},
  journal = {Bioinformatics},
  year = {2005},
  volume = {21},
  number = {9},
  pages = {1964-1970},
  doi = {http://dx.doi.org/10.1093/bioinformatics/bti287}
}
</pre></td>
</tr>
<tr id="Vinterbo2001" class="entry">
	<td>Vinterbo, S.A., Ohno-Machado, L. &amp; Dreiseitl, S.</td>
	<td>Hiding information by cell suppression <p class="infolinks">[<a href="javascript:toggleInfo('Vinterbo2001','bibtex')">BibTeX</a>]</p></td>
	<td>2001</td>
	<td>JAMIA<br/>Vol. Suppl. S, pp. 726-730&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Vinterbo2001" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Vinterbo2001,
  author = {Vinterbo, S. A. and Ohno-Machado, L. and Dreiseitl, S.},
  title = {Hiding information by cell suppression},
  journal = {JAMIA},
  year = {2001},
  volume = {Suppl. S},
  pages = {726--730}
}
</pre></td>
</tr>
<tr id="Vinterbo2012" class="entry">
	<td>Vinterbo, S.A., Sarwate, A.D. &amp; Boxwala, A.</td>
	<td>Protecting Count Queries in Study Design <p class="infolinks">[<a href="javascript:toggleInfo('Vinterbo2012','bibtex')">BibTeX</a>]</p></td>
	<td>2012</td>
	<td>J Am Med Inform Assoc<br/>Vol. 19(5), pp. 750-7&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1136/amiajnl-2011-000459">DOI</a> &nbsp;</td>
</tr>
<tr id="bib_Vinterbo2012" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Vinterbo2012,
  author = {Staal A. Vinterbo and Anand D. Sarwate and Aziz Boxwala},
  title = {Protecting Count Queries in Study Design},
  journal = {J Am Med Inform Assoc},
  year = {2012},
  volume = {19},
  number = {5},
  pages = {750-7},
  note = {PMID: 22511018},
  doi = {http://dx.doi.org/10.1136/amiajnl-2011-000459}
}
</pre></td>
</tr>
<tr id="sv:clust97" class="entry">
	<td>Vinterbo, S. &amp; &Oslash;hrn, A.</td>
	<td>A Rough Set Approach to Clustering <p class="infolinks">[<a href="javascript:toggleInfo('sv:clust97','bibtex')">BibTeX</a>]</p></td>
	<td>1997</td>
	<td><br/>Vol. 3Third Joint Conference Of Information Sciences, pp. 383-387&nbsp;</td>
	<td>inproceedings</td>
	<td><a href="self/Proceedings/RS-clustering.ea.pdf">URL</a>&nbsp;</td>
</tr>
<tr id="bib_sv:clust97" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inproceedings{sv:clust97,
  author = {S. Vinterbo and A. &Oslash;hrn},
  title = {A Rough Set Approach to Clustering},
  booktitle = {Third Joint Conference Of Information Sciences},
  year = {1997},
  volume = {3},
  pages = {383-387},
  url = {self/Proceedings/RS-clustering.ea.pdf}
}
</pre></td>
</tr>
<tr id="vinterbo:approximate99" class="entry">
	<td>Vinterbo, S. &amp; &Oslash;hrn, A.</td>
	<td>Minimal Approximate Hitting Sets and Rule Templates <p class="infolinks">[<a href="javascript:toggleInfo('vinterbo:approximate99','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>International Journal of Approximate Reasoning<br/>Vol. 25(2), pp. 123-143&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0888-613X(00)00051-7">DOI</a> <a href="http://www.sciencedirect.com/science/article/pii/S0888613X00000517">URL</a>&nbsp;</td>
</tr>
<tr id="bib_vinterbo:approximate99" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{vinterbo:approximate99,
  author = {S. Vinterbo and A. &Oslash;hrn},
  title = {Minimal Approximate Hitting Sets and Rule Templates},
  journal = {International Journal of Approximate Reasoning},
  year = {2000},
  volume = {25},
  number = {2},
  pages = {123--143},
  url = {http://www.sciencedirect.com/science/article/pii/S0888613X00000517},
  doi = {http://dx.doi.org/10.1016/S0888-613X(00)00051-7}
}
</pre></td>
</tr>
<tr id="Vinterbo2004:idamap" class="entry">
	<td>Vinterbo, S., Dreiseitl, S. &amp; Ohno-Machado, L.</td>
	<td>A Testing Procedure for htSNP Approximation Algorithms <p class="infolinks">[<a href="javascript:toggleInfo('Vinterbo2004:idamap','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Proceedings of IDAMAP 2004 at Stanford&nbsp;</td>
	<td>inproceedings</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_Vinterbo2004:idamap" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inproceedings{Vinterbo2004:idamap,
  author = {Staal Vinterbo and Stephan Dreiseitl and Lucila Ohno-Machado},
  title = {A Testing Procedure for htSNP Approximation Algorithms},
  booktitle = {Proceedings of IDAMAP 2004 at Stanford},
  year = {2004}
}
</pre></td>
</tr>
<tr id="sv:melanoma99" class="entry">
	<td>Vinterbo, S., Kittler, H., Seltenheim, M., Pehamberger, H. &amp; Binder, M.</td>
	<td>Computer aided image interpretation <p class="infolinks">[<a href="javascript:toggleInfo('sv:melanoma99','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Skin Research and Technology<br/>Vol. 5(2), pp. 155&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_sv:melanoma99" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sv:melanoma99,
  author = {S. Vinterbo and H. Kittler and M. Seltenheim and H. Pehamberger and M. Binder},
  title = {Computer aided image interpretation},
  journal = {Skin Research and Technology},
  year = {1999},
  volume = {5},
  number = {2},
  pages = {155},
  note = {ISSI'99 abstract}
}
</pre></td>
</tr>
<tr id="sv:multidiag99" class="entry">
	<td>Vinterbo, S. &amp; Ohno-Machado, L.</td>
	<td>A genetic algorithm approach to multidisorder diagnosis <p class="infolinks">[<a href="javascript:toggleInfo('sv:multidiag99','bibtex')">BibTeX</a>]</p></td>
	<td>2000</td>
	<td>Artificial Intelligence in Medicine<br/>Vol. 18(2), pp. 117-132&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/S0933-3657(99)00036-6">DOI</a> <a href="self/journal/gamdd.pdf">URL</a>&nbsp;</td>
</tr>
<tr id="bib_sv:multidiag99" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{sv:multidiag99,
  author = {S. Vinterbo and L. Ohno-Machado},
  title = {A genetic algorithm approach to multidisorder diagnosis},
  journal = {Artificial Intelligence in Medicine},
  year = {2000},
  volume = {18},
  number = {2},
  pages = {117--132},
  url = {self/journal/gamdd.pdf},
  doi = {http://dx.doi.org/10.1016/S0933-3657(99)00036-6}
}
</pre></td>
</tr>
<tr id="sv:recal99" class="entry">
	<td>Vinterbo, S. &amp; Ohno-Machado, L.</td>
	<td>A recalibration method for predictive models with dichotomous outcomes <p class="infolinks">[<a href="javascript:toggleInfo('sv:recal99','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>Predictive Models in Medicine: Some Methods for Construction and Adaptation<i>School</i>: Norwegian University of Science and Technology&nbsp;</td>
	<td>incollection</td>
	<td>&nbsp;</td>
</tr>
<tr id="bib_sv:recal99" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@incollection{sv:recal99,
  author = {S. Vinterbo and L. Ohno-Machado},
  title = {A recalibration method for predictive models with dichotomous outcomes},
  booktitle = {Predictive Models in Medicine: Some Methods for Construction and Adaptation},
  publisher = {Norwegian University of Science and Technology},
  school = {Norwegian University of Science and Technology},
  year = {1999},
  note = {ISBN 82-7984-011-7, ISSN 0802-6394}
}
</pre></td>
</tr>
<tr id="Vinterbo1999" class="entry">
	<td>Vinterbo, S. &amp; Ohno-Machado, L.</td>
	<td>A genetic algorithm to select variables in logistic regression: Example in the domain of myocardial infarction <p class="infolinks">[<a href="javascript:toggleInfo('Vinterbo1999','bibtex')">BibTeX</a>]</p></td>
	<td>1999</td>
	<td>JAMIA<br/>Vol. Suppl. S, pp. 984-988&nbsp;</td>
	<td>article</td>
	<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2232877/">URL</a>&nbsp;</td>
</tr>
<tr id="bib_Vinterbo1999" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Vinterbo1999,
  author = {Vinterbo, S. and Ohno-Machado, L.},
  title = {A genetic algorithm to select variables in logistic regression: Example in the domain of myocardial infarction},
  journal = {JAMIA},
  year = {1999},
  volume = {Suppl. S},
  pages = {984--988},
  url = {http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2232877/}
}
</pre></td>
</tr>
<tr id="sv:perffilt98" class="entry">
	<td>Vinterbo, S., Ohno-Machado, L. &amp; Fraser, H.</td>
	<td>A Description of a Strategy for Building Rough Set Classifiers Using Performance Filtering of Reducts <p class="infolinks">[<a href="javascript:toggleInfo('sv:perffilt98','bibtex')">BibTeX</a>]</p></td>
	<td>1998</td>
	<td>Proc. Sixth European Congress on Intelligent Techniques and Soft Computing (EUFIT'98), pp. 975-979&nbsp;</td>
	<td>inproceedings</td>
	<td><a href="self/Proceedings/eufit98.pdf">URL</a>&nbsp;</td>
</tr>
<tr id="bib_sv:perffilt98" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@inproceedings{sv:perffilt98,
  author = {S. Vinterbo and L. Ohno-Machado and H. Fraser},
  title = {A Description of a Strategy for Building Rough Set Classifiers Using Performance Filtering of Reducts},
  booktitle = {Proc. Sixth European Congress on Intelligent Techniques and Soft Computing (EUFIT'98)},
  year = {1998},
  pages = {975--979},
  url = {self/Proceedings/eufit98.pdf}
}
</pre></td>
</tr>
<tr id="Weber2004" class="entry">
	<td>Weber, G., Vinterbo, S. &amp; Ohno-Machado, L.</td>
	<td>Multivariate selection of genetic markers in diagnostic classification. <p class="infolinks">[<a href="javascript:toggleInfo('Weber2004','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Weber2004','bibtex')">BibTeX</a>]</p></td>
	<td>2004</td>
	<td>Artif Intell Med<br/>Vol. 31(2), pp. 155-67&nbsp;</td>
	<td>article</td>
	<td><a href="http://dx.doi.org/10.1016/j.artmed.2004.01.011">DOI</a> &nbsp;</td>
</tr>
<tr id="abs_Weber2004" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Analysis of gene expression data obtained from microarrays presents a new set of challenges to machine learning modeling. In this domain, in which the number of variables far exceeds the number of cases, identifying relevant genes or groups of genes that are good markers for a particular classification is as important as achieving good classification performance. Although several machine learning algorithms have been proposed to address the latter, identification of gene markers has not been systematically pursued. In this article, we investigate several algorithms for selecting gene markers for classification. We test these algorithms using logistic regression, as this is a simple and efficient supervised learning algorithm. We demonstrate, using 10 different data sets, that a conditionally univariate algorithm constitutes a viable choice if a researcher is interested in quickly determining a set of gene expression levels that can serve as markers for disease. We show that the classification performance of logistic regression is not very different from that of more sophisticated algorithms that have been applied in previous studies, and that the gene selection in the logistic regression algorithm is reasonable in both cases. Furthermore, the algorithm is simple, its theoretical basis is well established, and our user-friendly implementation is now freely available on the internet, serving as a benchmarking tool for the development of new algorithms.</td>
</tr>
<tr id="bib_Weber2004" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Weber2004,
  author = {Griffin Weber and Staal Vinterbo and Lucila Ohno-Machado},
  title = {Multivariate selection of genetic markers in diagnostic classification.},
  journal = {Artif Intell Med},
  year = {2004},
  volume = {31},
  number = {2},
  pages = {155-67},
  doi = {http://dx.doi.org/10.1016/j.artmed.2004.01.011}
}
</pre></td>
</tr>
<tr id="Weber2002" class="entry">
	<td>Weber, G., Vinterbo, S. &amp; Ohno-Machado, L.</td>
	<td>Building an asynchronous web-based tool for machine learning classification. <p class="infolinks">[<a href="javascript:toggleInfo('Weber2002','abstract')">Abstract</a>] [<a href="javascript:toggleInfo('Weber2002','bibtex')">BibTeX</a>]</p></td>
	<td>2002</td>
	<td>JAMIA<br/>Vol. Suppl. S, pp. 869-73&nbsp;</td>
	<td>article</td>
	<td>&nbsp;</td>
</tr>
<tr id="abs_Weber2002" class="abstract noshow">
	<td colspan="6"><b>Abstract</b>: Various unsupervised and supervised learning methods including support vector machines, classification trees, linear discriminant analysis and nearest neighbor classifiers have been used to classify high-throughput gene expression data. Simpler and more widely accepted statistical tools have not yet been used for this purpose, hence proper comparisons between classification methods have not been conducted. We developed free software that implements logistic regression with stepwise variable selection as a quick and simple method for initial exploration of important genetic markers in disease classification. To implement the algorithm and allow our collaborators in remote locations to evaluate and compare its results against those of other methods, we developed a user-friendly asynchronous web-based application with a minimal amount of programming using free, downloadable software tools. With this program, we show that classification using logistic regression can perform as well as other more sophisticated algorithms, and it has the advantages of being easy to interpret and reproduce. By making the tool freely and easily available, we hope to promote the comparison of classification methods. In addition, we believe our web application can be used as a model for other bioinformatics laboratories that need to develop web-based analysis tools in a short amount of time and on a limited budget.</td>
</tr>
<tr id="bib_Weber2002" class="bibtex noshow">
<td colspan="6"><b>BibTeX</b>:
<pre>
@article{Weber2002,
  author = {Griffin Weber and Staal Vinterbo and Lucila Ohno-Machado},
  title = {Building an asynchronous web-based tool for machine learning classification.},
  journal = {JAMIA},
  year = {2002},
  volume = {Suppl. S},
  pages = {869-73}
}
</pre></td>
</tr>
</tbody>
</table>

<p>
 <small>Created by <a href="http://jabref.sourceforge.net">JabRef</a> on 10/03/2015.</small>
</p>

</body>
</html>

<!-- File generated by JabRef ; Export Filter written by Mark Schenk -->